README.md

```diff
@@ -37,10 +37,10 @@ Backend binaries are output to `./bin`.
 
 The following instructions assume you have an existing cluster you can connect
 to. OpenShift 4.x clusters can be installed using the
-[OpenShift Installer](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
-You can also use [CodeReady Containers](https://github.com/code-ready/crc)
-for local installs. More information about installing OpenShift can be found at
+[OpenShift Installer](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/). More information about installing OpenShift can be found at
 <https://try.openshift.com/>.
+You can also use [CodeReady Containers](https://github.com/code-ready/crc)
+for local installs, or native Kubernetes clusters.
```

Comment:
> or native Kubernetes clusters

Technically this is true, but we've not done a good job of testing against vanilla Kube, so I'm not sure we want to highlight this.  Your thoughts, @spadgett?

Reply:
thanx @rhamilto 

to clarify this change - the content is not new, it's just rearranging the paragraph so relevant content is grouped together (bring the openshift sentences close by).

if we want to drop the declarations about vanilla kube, we should handle it throughout the file, and take it to a separate PR.

Reply:
Ah.  Thanks for the clarification, @eliranmal.